src/pages/events/CreateEvent.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState, useEffect } from 'react';
+import {
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+  Button,
+  VStack,
+  Box,
+  useToast,
+} from '@chakra-ui/react';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import { useDispatch } from 'react-redux';
+import { createEvent } from '../../redux/slice/eventSlice';
+
+const EventForm = () => {
```

Comment:
- [ ] When I successfully create an event, the app tells me that it has been reserved, not created:

![image](https://github.com/RitaDee/event_connect_client/assets/57421823/cc666ae6-a517-45ef-a59e-8b351c94c3b3)

Kindly change that word from "reserved" to "created." 👍